repo/fsrepo/migrations/migrations.go

```diff
@@ -155,13 +155,14 @@ func ReadMigrationConfig(repoRoot string) (*config.Migration, error) {
 // downloadSources,
 func GetMigrationFetcher(downloadSources []string, distPath string, newIpfsFetcher func(string) Fetcher) (Fetcher, error) {
 	const httpUserAgent = "go-ipfs"
+	const numRetriesPerHTTP = 3
```

Comment:
Just made up a number here, no delays, etc. Again not sure we actually need/want this here